src/NuGet.Core/NuGet.Configuration/PackageNamespaces/PackageNamespacesConfiguration.cs

```diff
@@ -14,9 +14,23 @@ public class PackageNamespacesConfiguration
         /// </summary>
         public Dictionary<string, IReadOnlyList<string>> Namespaces { get; }
```

Comment:
I am not sure if this property is really required here because of following reasons.

- As of now there is no use case that reads `Namespaces` property.
- All we need is an easy lookup that Trie structure offers.
Hence, I think `PackageNamespacesConfiguration` type can only have 1 public property pointing to the Trie structure is sufficient.

Reply:
@nkolev92 
Any thoughts? Do we need it?

Reply:
yeah, I agree, we can remove that for now.

Reply:
After checking maybe it's better not to remove this. Because it's useful for correctness checks and there're 5 unit tests for it. I don't think SearchTree/Trie can replace it because it's already combined/transformed data.

Reply:
@nkolev92 
Can you elaborate more on `consider making it for now`? Is it keep or remove it?

Reply:
Ok. Addressed.